identity/README.md

```diff
@@ -8,27 +8,35 @@ such as users, roles, permissions, and OAuth clients.
 
 ## Running locally
 
-1. Build project from the root folder of the monorepo using:
+- All commands should be run from the root folder of the monorepo unless stated otherwise.
+
+1. Build project:
 
 ```
-mvn clean install -U --projects dist --also-make -DskipTests -Dskip.fe.build=false
+mvn clean install -DskipTests=true -DskipChecks -Dskip.fe.build=false -DskipQaBuild -T1C
 ```
 
 2. Run elasticsearch by using one of the already configured docker-compose file in our codebase.
 
-Example: navigate to Operate's folder in the codebase and run:
+Example (using Operate's configuration):
 
 ```
-docker compose -f config/docker-compose.yml up -d elasticsearch
+docker compose -f operate/docker-compose.yml up -d elasticsearch
 ```
 
-3. Run this command for some final setup:
+3. Run this command for some final setup and running the Zeebe broker:
 
 ```
-CAMUNDA_SECURITY_AUTHORIZATIONS_ENABLED=true \
+SPRING_PROFILES_ACTIVE=consolidated-auth,broker,identity \
```

Comment:
We can potentially add instruction of how to add operate/tasklist by adding the extra profiles

Reply:
@OmranAbazid I'm not sure how to do that, would it be just adding `operate` or `tasklist` in the profiles list on this command?

If you know also feel free to push a commit to this PR with that change or DM me and we can do it together if you'd prefer, the idea is to make people lives easier with this step-by-step, so this PR is super open to be collaborative.

Reply:
I do not mind having them always set or at least a comment below the command saying:
> If you would like to enable operate and tasklist, you can add their profile to the `SPRING_PROFILES_ACTIVE` variable like this:
```
SPRING_PROFILES_ACTIVE=operate, tasklist, consolidated-auth,broker,identity
```